### ISSUE TITLE
Composer dependency "text_languagedetect" out of support

### ISSUE THREAD

**Issue by nupplaphil** [friendica/friendica#9348]
- [x] I have searched open and closed issues for duplicates

### Bug Description

The composer dependency `pear/text_languagedetect` is out of support, see
https://pear.php.net/package/Text_LanguageDetect

Therefore no new bugfixings will be added and in case of composer 2.0, the library will brick (like the composer notices already say):
> Deprecation Notice: Class Text_LanguageDetect_ISO639Test located in ./vendor/pear/text_languagedetect/tests/Text_LanguageDetect_ISO639Test.php does not comply with psr-0 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///home/philipp/PhpstormProjects/friendica/bin/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:201
> Deprecation Notice: Class Text_LanguageDetectTest located in ./vendor/pear/text_languagedetect/tests/Text_LanguageDetectTest.php does not comply with psr-0 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///home/philipp/PhpstormProjects/friendica/bin/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:201

### Steps to Reproduce

Install a new Friendica node an run `php bin/composer.phar install`

Actual Result:

composer notices

Expected Result:

no composer problems and further maintenance of our dependency library.

### Platform Info

Friendica Version: 2020.12-dev

Friendica Source: git

PHP version: 7.3

SQL version:

**Comment by annando** [comment]
Is there some alternative? Sadly the one that @MrPetovan found is based upon this as well, I think:
```
$ bin/composer.phar require peterkahl/language-genius —no-update
$ bin/composer.phar update peterkahl/language-genius —no-dev
```

**Comment by annando** [comment]
I guess we can now close this.

**Comment by MrPetovan** [comment]
Closed by #9350

**Comment by MrPetovan** [comment]
Part of #8796
